server/middleware/gqlAuth/tests/index.test.js

```diff
@@ -186,107 +186,107 @@ describe('gqlAuth tests', () => {
       expect(response).toBe(true);
     });
   });
-  describe('isAuthenticated tests', () => {
-    let mocks;
-    beforeEach(() => {
-      const spy = jest.fn();
-      mocks = {
-        next: spy
-      };
-      process.env.ENVIRONMENT_NAME = 'dev';
-      process.env.NODE_ENV = 'dev';
-    });
-    const req = {
-      body: {
-        query: `query Address{
-          address(id:1){
-            id
-          }
-        }
-        `
-      },
-      headers: {}
-    };
+  // describe('isAuthenticated tests', () => {
```

Comment:
letts cleanup

Reply:
we don't want to merge commented code. Let's clean all of it

Reply:
roger